testdata/valid-docs/article.json

```diff
@@ -0,0 +1,127 @@
+{
+  "uuid": "98cedf81-ce8d-4dc4-bcbc-6091a756cc5c",
+  "type": "core/article",
+  "uri": "core://article/98cedf81-ce8d-4dc4-bcbc-6091a756cc5c",
+  "title": "champ-inter",
+  "content": [
+    {
+      "id": "NzUsMjQ2LDIwOCwxNTE",
+      "type": "core/heading-1",
```

Comment:
Is this really a valid article?

Reply:
Sorry, forgot this one, what specifically did you react to? That the type is "core/heading-1"?